src/views/v2/event-status/postponed-label.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Reason container for a postponed event.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/events-status/postponed-label.php
```

Comment:
if we are following the folder structure we've used for the other views, this should live in: `tribe/events/v2/events-status/status-label/postponed-label.php`